osc-dm-mesh-cli/src/utl/product_generator.py

```diff
@@ -135,11 +135,9 @@ def create_artifact_file(
     contents["artifact"]["name"] = name
     contents["artifact"]["links"][0]["url"] = data_url
 
-
+    print(f"artifact type: {artifact_type}")
```

Comment:
this should either be deleted (if it is something you used to test, and no longer need), or should be converted to a log statement instead of print.

Reply:
removed